### PR TITLE
fix passphrase recovery if no username is passed

### PR DESCRIPTION
**Pull request by pzduniak**

### CODE REVIEW

File: go/engine/passphrase_recover.go [hunk 1]
```diff
@@ -234,6 +243,7 @@ func (e *PassphraseRecover) suggestReset(mctx libkb.MetaContext) (err error) {
 
 	// We are certain the user will not know their password, so we can disable that prompt.
 	eng := NewAccountReset(mctx.G(), e.arg.Username)
+	mctx.Info("huh %s", e.arg.Username)
```
Comment (zapu) [inline go/engine/passphrase_recover.go]: :scissors:

**Comment by zapu** [comment]
`keybase passphrase recover` already asks for username, also coming from LoginUI, but the prompt doesn't seem to do anything, How is it triggered now?

File: go/engine/passphrase_recover.go (hunk 1 above)
Comment (pzduniak) [reply go/engine/passphrase_recover.go]: oops

**Comment by pzduniak** [comment]
Right now loaduser triggers it, it uses the exact same code as I added here. This engine wasn't supposed to work without a passed username, this PR fixes it.

**Pull request merged**